src/main/scala/cromwell/binding/values/WdlArray.scala

```diff
@@ -25,6 +25,7 @@ case class WdlArray(wdlType: WdlArrayType, value: Seq[WdlValue]) extends WdlValu
   def tsvSerialize: Try[String] = {
     wdlType.memberType match {
       case t: WdlPrimitiveType => Success(value.map(_.valueString).mkString("\n"))
+      case WdlObjectType => WdlObject.tsvSerializeArray(value map { _.asInstanceOf[WdlObject] })
```

Comment:
This is done in the `tsvSerializeArray` function (there is a `attributesMatch` function)